docs/topics/modules/planning_development_work.adoc

```diff
@@ -0,0 +1,91 @@
+[id="planning_development_work"]
+= Planning your development work
+
+After creating a space, use the {osio} planner to plan your development work and to ensure efficient execution of your project.
+
+NOTE: The development process you select, while creating the space, determines the planner's guided work item type hierarchy.
+
+This example walks you through the Scenario Driven Development planning process to plan an end to end development flow of launching and deploying a sample application.
+
+. In the dashboard of your space, select the *Plan* tab at the top of the page to view the planner. The default *Backlog* view displays the *Scenarios* work item type group for your space.
+. Select the *Show Tree* check box to see the work items in a tree view. The tree view allows you to quickly add work items according to the guided hierarchy using the quick-add and in-line add options.
+. Use the quick-add option to create a *Scenario* as follows:
+.. In the *Add Work Item* pane, select *Scenario* from the drop-down list.
++
+image::quickadd_wi_type.png[Quick Add work item type]
++
+.. In the *Work Item Title* field, add `Enable an end to end, quick, developer work-flow, within a continuous integration and deployment model` as the title.
+.. Press kbd:[Enter] to create the scenario.
++
+image::gs_scenario_eg.png[Example Scenario]
++
+
+. Similarly, select *Fundamental* in the *Add Work Item* pane, to create a fundamental work item with the title `The application should be secure with appropriate licenses`.
```

Comment:
@Preeticp just IMO, if you look at the build preview, all the `` rendering for long work item titles looks a bit iffy. I'd recommend just plain bold to prevent the monospace rendering.

Reply:
I know, they render red, not exactly great, was just following the convention of putting any user input text in monospace. Once they are entered, in the following steps I have added them in Bold.